scripts/dbsize.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+df --output=pcent "$1" | tr -dc ‘0-9’
```

Comment:
This requires GNU coreutils' version of `df`, so in order to at least also make this work on macOS (which I would much appreciate for testing actually), please change it like so:

```bash
PATH="$(brew --prefix coreutils)/libexec/gnubin:${PATH}" df --output=pcent . | tr -dc '0-9'
```

(Or make it a bit smarter to not always modify $PATH, but instead to only do so if `brew --prefix coreutils` doesn't fail.)